Profiles/cio-Condition.xml

```diff
@@ -52,24 +52,19 @@
       <path value="Condition.extension" />
       <min value="1" />
     </element>
-    <element id="Condition.extension:conditionStatusDate">
+    <element id="Condition.extension:StatusDate">
```

Comment:
Slice names are in camelCase (see https://informatiestandaarden.nictiz.nl/wiki/FHIR:V1.0_FHIR_Profiling_Guidelines_R4#Slice_names), hence it should be 'statusDate'.